cli/dstack/cli/updates.py

```diff
@@ -0,0 +1,74 @@
+import os
+import uuid
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+import requests
+from packaging import version as pkg_version
+
+from dstack import version
+
+
+def anonymous_installation_id() -> str:
+    ai_id_path = Path.home() / ".cache" / ".dstack" / "anonymous_installation_id"
```

Comment:
Why storing metadata files in `.cache/.dstack` and not in `.dstack`? We already have our place in home dir.